packages/react-components/src/radio/src/RadioGroup.tsx

```diff
@@ -120,6 +124,9 @@ export function InnerRadioGroup(props) {
         disabled,
         children,
         forwardedRef,
+        // we don't want size passed down to Group, since
```

Comment:
size used to be in the rest property, but nobody down the line supports this props, we pass it down to a Box, whos passes it down to a div, but its not supported there.

Reply:
Why size would be in the "rest" prop? Pretty sure a RadioGroup or Radio doesn't support `size`.

Reply:
My fix was a bad one. 

the correct fix would be 
```
const {
        value,
        defaultValue,
        required,
        validationState,
        name,
        onChange,
        autoFocus,
        orientation = "vertical",
        gap,
        wrap,
        reverse,
        disabled,
        children,
        forwardedRef,
        ...rest
    } = mergeProps(
        props,
        toolbarProps,
        omitProps(fieldProps, ["fluid", "size"])  <---- fieldProps contains a size, but size is not supported by <Group
    );
```

Reply:
Sounds good